### DISCUSSION TITLE
Pointing subdomain to an IP address

### DISCUSSION THREAD

**Discussion by ackledotdev** [thedev-id/thedev.id#462] [Q&A]
Hi there! I'm hosting an Express.js web server on a Raspberry Pi, and I'm wondering if I could get a subdomain and point it to my computer. Would that be possible?
If not, is there a way for me to point my github.io subdomain to my Pi?
Thanks in advance!

**Comment by lakuapik** [comment]
@akpi816218 do you still want to have it? i can help you set it up. and does your webserver facing public ip? or local ip?

**Reply by ackledotdev** [reply]
I was just wondering if it would be possible. I'm not looking for a subdomain right now. Thanks though!
